intermine_boot/intermine_docker.py

```diff
@@ -219,19 +233,20 @@ def create_intermine_builder_container(client, image, env, options):
     data_dir = env['data_dir'] / 'docker' / 'data'
 
     environment = {
-        'MINE_NAME': os.environ.get('MINE_NAME', 'biotestmine'),
+        'MINE_NAME': _get_mine_name(options),
         'MINE_REPO_URL': os.environ.get('MINE_REPO_URL', ''),
         'MEM_OPTS': os.environ.get('MEM_OPTS', '-Xmx2g -Xms1g'),
-        'IM_DATA_DIR': os.environ.get('IM_DATA_DIR', '')
+        'IM_DATA_DIR': os.environ.get('IM_DATA_DIR', ''),
+        'IM_DATA_FROM_PATH': options['datapath_im'] if options['datapath_im'] else ''
```

Comment:
I wonder if we should make the naming of this envvar clearer.

Looking at https://github.com/intermine/docker-intermine-gradle/pull/11, the path is used like a boolean, to force it to build from scratch even though the mine folder has files in it.

How about renaming it to `FORCE_MINE_BUILD` and passing "true" as the value instead?

We also still want to be able to start a mine we have previously built, without building it from scratch again. You can achieve this by editing `_is_conf_same` to also save `datapath_im`, then use the result of the function to determine whether `FORCE_MINE_BUILD` should be true or empty.

Reply:
> Looking at intermine/docker-intermine-gradle#11, the path is used like a boolean, to force it to build from scratch even though the mine folder has files in it.
Yes.

> How about renaming it to FORCE_MINE_BUILD and passing "true" as the value instead?
ok, FORCE_MINE_BUILD sounds good. 

>We also still want to be able to start a mine we have previously built, without building it from scratch again. You can achieve this by editing _is_conf_same to also save datapath_im, then use the result of the function to determine whether FORCE_MINE_BUILD should be true or empty.
Right.

Reply:
done.